### DISCUSSION TITLE
Can I save some values separate from others?

### DISCUSSION THREAD

**Discussion by storyteller-dev** [ctreffe/alfred#136] [Alfred Q&A] [answered]
I'd like to raffle a coupon as a reward for participating in my experiment. 
Ideally, the email address would be stored separately from all other experiment data, so it will still be completely anonymous. How can I achieve that?

**Comment by jobrachem** [comment, marked as answer]
Alfred3 offers the `alfred3.UnlinkedDataPage` ([docs](https://alfredo3.psych.bio.uni-goettingen.de/docs/generated/alfred3.page.UnlinkedDataPage.html?highlight=unlinkeddata)) page exactly for this purpose. You can simply use it as an alias for `alfred3.Page`. All data entered to input elements on an UnlinkedDataPage will be saved separately from the remaining experiment data and shuffled upon export. The unlinked data set does not contain identifying session info.

Here's an example: 

```python
import alfred3 as al
exp = al.Experiment()

@exp.member
class Demo(al.UnlinkedDataPage):

    def on_exp_access(self):
        self += al.TextEntry(toplab="Enter here", name="el1")
```
